content-repo/extra-docs/articles/troubleshooting.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Troubleshooting Guide
+description: This article helps common troubleshooting issues with XSOAR integrations
+---
+
+# Troubleshooting Guide
+
+##Connection issues troubleshooting guide:
+### When a customer has any connection errors try the following steps:
+1. Check and uncheck the “insecure” checkbox
+2. Check and uncheck the “proxy” checkbox
+3. Try to isolate the problem, is it the integration instance server or the client?
+
+    a. In case you have access from your local computer to the integration instance server, Try to generate the same request using curl or postman from your local computer (or customer’s computer)- if it fails: the integration instance server is probably down, could be instance issue.
+    
+    b. Try to do some curl from the XSOAR machine, for example: 
+    ```
+   curl https://httpbin.org/status/200  -vk
+   ```
+    and see if the returning status is 200
+
+    c. Try to run some curl request via a docker container using 
+    ```
+   docker run -it demisto/netutils:1.0.0.6138 curl https://httpbin.org/status/200  -vk
```

Comment:
need to mention that docker is an integration related value so they need to use the proper one.
in the command example, I would use the default docker image.

Reply:
The default container does not have curl installed